### PR TITLE
Add a content_type query switch to the CLI.

### LINKED ISSUE #1328: Modify the CLI to work with multiple content types
We will need to modify the CLI to handle multiple content types, once #1327 is done.

### PR DESCRIPTION
**Pull request by ralphbean**
Fixes #1328 (and complements #1581).

### CODE REVIEW

File: bodhi/client/__init__.py [hunk 1]
```diff
@@ -236,6 +236,8 @@ def edit(user, password, url, **kwargs):
               help='Query only critical path packages')
 @click.option('--cves', help='Query by CVE id')
 @click.option('--packages', help='Query by package name(s)')
+@click.option('--content-type', help='Query updates based on content type',
+              type=click.Choice(['rpm', 'module']))  # And someday, container.
```
Comment (pypingou) [inline bodhi/client/__init__.py:240]: Should it default to rpm?

**Comment by pypingou** [comment]
One question, otherwise 👍 for me

File: bodhi/client/__init__.py (hunk 1 above)
Comment (ralphbean) [reply bodhi/client/__init__.py:240]: I just don't know, but am open to opinions.

By defaulting to `None`, a query for `--packages=nginx` would return all updates with the nginx package and those with the nginx module.

Comment (pypingou) [reply bodhi/client/__init__.py:240]: Sounds also sane, let's see what @bowlofeggs prefers

Comment (ralphbean) [reply bodhi/client/__init__.py:240]: Hm, a thought.  If the default is anything other than `None` (e.g., if it is 'rpm'), then there is no way to query for *all* updates of *any* content type.

Comment (bowlofeggs) [reply bodhi/client/__init__.py:240]: Yeah I think I like letting it default to showing all content types.

**Review by bowlofeggs — APPROVED** [review]
Seems to work!

**Pull request merged**